webapp/src/modules/parcels/reducer.js

```diff
@@ -80,15 +80,23 @@ export function parcelsReducer(state = INITIAL_STATE, action) {
     }
     case FETCH_PUBLICATIONS_SUCCESS: {
       const { assets } = action
-      const parcels = assets.filter(asset => isParcel(asset))
+
+      const data = { ...state.data }
+
+      for (const asset of assets) {
+        if (isParcel(asset)) {
+          data[asset.id] = {
+            ...data[asset.id],
+            ...normalizeParcel(asset)
```

Comment:
Shouldn't we need to keep using `toParcelObject` here?

Reply:
I didn't use it because `toParcelObject` takes an array as param and I need to add a single asset to a map of `{id: parcel, ...}`